data-model/SDP-data-model-templates/textual-artifact_sdp-template.json

```diff
@@ -147,15 +165,26 @@
     "viewing_hint": "CV",
     "range": "???"
   },
-  "contributor": [
-    "array of strings (CV)"
-  ],
-  "related_objects": [
-    {
-      "relationship_type": "CV",
-      "object": "ARK",
-      "position": "int",
-      "layer": "int"
-    }
-  ]
+  "related_objects": {
+    "contained_in": [
```

Comment:
I didn't like 'content_of' as the reciprocal but am having trouble coming up with good language here.